### DISCUSSION TITLE
Questioning the political positioning of the composer tool

### DISCUSSION THREAD

**Discussion by johnkhansrc** [composer/composer#11912] [General]
Hello,

I wanted to propose this because last time in https://github.com/composer/composer/issues/11707 and https://github.com/composer/composer/issues/11865 the situation was considered "much more complex"

But now, the situation of apartheid and genocide is recognized by the UN (United Nations Organization).

Personally I am against the political positioning in the composer tool, however we still have #StandWithUkraine, so there is an ethical trade-off to be made if we do not want to be considered as supporting the genocide in progress.

I propose that we remove all political quotes from the tool, or that we expand it to the reality of the geopolitical scene

**Comment by Seldaek** [comment]
#StandWithUkraine has been removed too so I am not sure what you are referring to, but yeah I think we should refrain from further political commentary.
